core/src/style/textStyle.cpp

```diff
@@ -67,12 +70,15 @@ Parameters TextStyle::parseRule(const DrawRule& _rule) const {
     _rule.get(StyleParamKey::transform, transform);
     _rule.get(StyleParamKey::visible, p.visible);
     _rule.get(StyleParamKey::priority, p.priority);
+    if (_rule.get(StyleParamKey::text_source, p.textSource.text)) {
+        p.textSource.isFunction = _rule.isJSFunction(StyleParamKey::text_source);
+    }
 
-    if (transform == "capitalize") {
+    if (transform == capitalize) {
         p.transform = TextTransform::capitalize;
-    } else if (transform == "lowercase") {
+    } else if (transform == lowercase) {
         p.transform = TextTransform::lowercase;
-    } else if (transform == "uppercase") {
+    } else if (transform == uppercase) {
         p.transform = TextTransform::uppercase;
     }
```

Comment:
string== has an overload for char* comparison, so here the heap allocated static strings wouldn't be needed. But I would like to move the transform string-to-enum conversion to StyleParam::parseString.